### PR TITLE
MDEXP-327 Bugfix release 3.0.1

### PR DESCRIPTION
**Pull request by andrii-novytskyi**
https://issues.folio.org/browse/MDEXP-327

## Purpose
Bugfix release to handle csv file with different symbols, handle empty csv file correctly,
displaying UUIDs of records from inventory or SRS that were not found in error logs,fixed missing $ 3 subfield in MARC
file and duplication of multiple error log entries. Also, this release includes migration script for 
mapping profiles.

**APPROACH**
1. Created branch tmp-release-3.0.1
2. Cherry pick commits:  a08d93f710e4cd547f5b4680fb4f4a060572da7a, 834077dbb22cabcd030e596dc29e56f8579b1e2a,
5bc801f1181e3fd5af6d2dc83d50e3fba0c671ca, 11019ecc3fb58cadf225cc800cb558c8b8675a9c, a33042c28dfc7c0d313a7852c05aca1bc2d74679, 78335e20806475df834c3c85538cafc9983ff16a, 6de37b3b479c8d9b6fbb4e1ec952617cc4a69a22, 304292497b84de32dcc2d076f1ce79ddcd40d49e, 83510f5f3375c452d9d09994a94c75ed6350b52f, ea47517ef4ae52d8ff0cc0ec716eded3b1d197b9, bf63b623ed1bb91ab3af13c9231600ed23941ee3, 7db3a4df1320f02e58c634796687fa27de4a49fe, 8680955df5e981b391f907d95bcfdd8c209f1759.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+### Bug Fixes
+* [MDEXP-176](https://issues.folio.org/browse/MDEXP-176) - Handle csv files with other characters
+* [MDEXP-267](https://issues.folio.org/browse/MDEXP-267) - After uploading of empty file the progress field is not provided
+* [MDEXP-305](https://issues.folio.org/browse/MDEXP-305) - Investigate if generated mrc files and directories are not being deleted
+* [MDEXP-307](https://issues.folio.org/browse/MDEXP-307) - Fix security dependency issue
+* [MDEXP-308](https://issues.folio.org/browse/MDEXP-308) - Subfield $3 is missing for the MARC tags with item data
+* [MDEXP-309](https://issues.folio.org/browse/MDEXP-309) - When UUID is not found the error log doesn't list it
+* [MDEXP-313](https://issues.folio.org/browse/MDEXP-313) - Failed job associated with generated mrc file and contains multiple entries in the error log
+* [MDEXP-314](https://issues.folio.org/browse/MDEXP-314) - Mapping profile migrate fieldIds
+* [MDEXP-316](https://issues.folio.org/browse/MDEXP-316) - Error logs does not contains items records if mapping process is failed
```
Comment (KVupp) [inline NEWS.md:18]: we usually add them in the reverse order. so 316 goes up. You can use excel sort to do it easily

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     },
     {
       "id": "users",
-      "version": "15.2"
+      "version": "15.3"
```
Comment (KVupp) [inline descriptors/ModuleDescriptor-template.json:11]: The latest release is still at 15.2. Please revert this:https://github.com/folio-org/mod-users/blob/v17.2.2/descriptors/ModuleDescriptor-template.json#L7

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+
+
```
Comment (KVupp) [inline NEWS.md:20]: https://issues.folio.org/browse/MDEXP-317 this is missing

Comment (KVupp) [reply NEWS.md:20]: https://issues.folio.org/browse/MDEXP-311 Even this

Comment (KVupp) [reply NEWS.md:20]: nvm.. we can ignore 317, it's just generate-marc-utils PR.

File: NEWS.md (hunk 1 above)
Comment (andrii-novytskyi) [reply NEWS.md:10]: updated

File: NEWS.md (hunk 3 above)
Comment (andrii-novytskyi) [reply NEWS.md:20]: added MDEXP-311

**Review by KVupp — APPROVED** [review]

**Review by Igor-Gorchakov — APPROVED** [review]

**Pull request merged**